docs/faq.rst

```diff
@@ -363,13 +363,10 @@ Overall training flow related questions
 Known issues
 ************
 
-#. If server dies and then is restarted, intentionally or unintentionally, all clients will have to be restarted.
 #. Running out of memory can happen at any time, especially if the server and clients are running on same machine.
    This can cause the server to die unexpectedly.
-#. Putting applications in the transfer folders without using the upload_app command or forgetting to delete the models
-   folder inside, a mysterious error may occur when running the deploy_app command because the application folder is too
-   large to be uploaded and that causes timeout.
-#. Please don't start a new training run or start a new app before the previous application is fully stopped. Users
-   can do ``abort client`` and ``abort server`` before ``start_app`` for the new run.
 #. After calling ``shutdown client`` for a client running multi GPUs, a process (sub_worker_process) may remain. The
    work around for this is to run ``abort client`` before the ``shutdown`` command.
+#. If a snapshot is in a corrupted state, the server may try to restore the job and get stuck. To resolve this, delete
+   the snapshot from the location configured in project.yml for the snapshot_persistor storage, and ``abort_job`` should
+   be able to stop the job on the server.
```

Comment:
maybe we need to provide an example to this, like `/tmp/nvflare/jobs-storage`.

Or we need to explain these in the page that contains the default project.yml
Our SAs and QAs also have questions about what these locations mean.